README.md

```diff
@@ -6,100 +6,119 @@ Auto-generated [D](https://dlang.org) bindings for the [sokol headers](https://g
 
 #### Targets
 
-- Native
-- Wasm (`-Dtarget=wasm32-emscripten-none`) - LTO enabled on release-mode.
-
 By default, the backend 3D API will be selected based on the target platform:
 
 - macOS: Metal
 - Windows: D3D11
-- Linux: GL
+- Linux: GL/Wayland
+- Wasm (LDC2 required) - WebGL3/WebGPU support
+
+#### Usage - template project
+
+```console
+dub init -t sokol-d
+```
+
+## Build Requirements
+
+- [D Compiler](https://dlang.org/download.html) (DMD/LDC2/GDC) v2.111.0 or higher
+- C/C++ compiler (GCC/Clang/MSVC)
+- Linux dependencies: libglu1-mesa-dev, mesa-common-dev, xorg-dev, libasound-dev (for X11/Wayland, GL and ALSA development)
+
+## Building Native Examples
+
+1. First, build the sokol static library:
+```bash
+dub build -c sokol-static
+```
 
-## BUILD
+2. Then build and run any example:
+```bash
+cd examples
+dub :triangle -c native
+dub :cube -c native
+dub :imgui -c native
+```
 
-**Required**
+## Building WebAssembly Examples
 
-- [zig](https://ziglang.org/download) v0.14.0 or master
-- [ldc](https://ldc-developers.github.io) v1.40.0 or latest-CI (nightly)
+1. First, build sokol for WebAssembly:
+```bash
+# For WebGL2
+dub build -c sokol-wasm-webgl
 
-Supported platforms are: Windows, macOS, Linux (with X11)
+# For WebGPU
+dub build -c sokol-wasm-wgpu
+```
```

Comment:
I think the webgl and webgpu build instructions are still wrong here? The build configs `sokol-wasm-webgl` and `sokol-wasm-wgpu` don't seem to exist (instead `sokol-wgles3` and `sokol-wgpu`).